frontend/src/components/Share/Share.js

```diff
@@ -1,100 +1,59 @@
-import React from 'react';
-import './Share.scss';
+import React, { useState, useRef } from "react";
+import "./Share.scss";
 import share from "../../assets/share.svg";
 
-class Share extends React.Component {
-  constructor() {
-    super();
-
-    this.handleClick = this.handleClick.bind(this);
-    this.handleOutsideClick = this.handleOutsideClick.bind(this);
-
-    this.state = {
-      showModal: false,
-      copySuccess: false,
-
-    };
-  }
-
-  handleClick() {
-    if (!this.state.showModal) {
-      // attach/remove event handler
-      document.addEventListener('click', this.handleOutsideClick, false);
-    } else {
-      document.removeEventListener('click', this.handleOutsideClick, false);
-    }
-
-    this.setState(prevState => ({
-       showModal: !prevState.showModal,
-    }));
-  }
-
-  handleOutsideClick(e) {
-    // ignore clicks on the component itself
-    if (this.node.contains(e.target)) {
-      return;
-    }
-
-    this.handleClick();
-  }
-
-   copyCodeToClipboard = () => {
-    this.urlRef.select()
-    document.execCommand("copy")
-    this.setState({copySuccess: true})
-      setTimeout(() => {
-          this.setState(prevState => ({
-       copySuccess: !prevState.copySuccess,
-       showModal: !prevState.showModal,
-    }))
-         }, 3000);
-
+function Share() {
+
+  const [modalShow, setModalShow] = useState(false);
+  const [copySuccess, setcopySuccess] = useState(false);
+  const message = "Share Covexit with others and help during this crisis.";
+  const inputRef = useRef(null);
```

Comment:
no need for `null`

Reply:
I can't seem to get it to work if I remove null